include/swift/AST/DiagnosticsSema.def

```diff
@@ -3857,6 +3857,10 @@ WARNING(redundant_particular_literal_case,none,
 NOTE(redundant_particular_literal_case_here,none,
      "first occurrence of identical literal pattern is here", ())
 
+ERROR(cannot_prove_exhaustive_switch,none,
```

Comment:
@jrose-apple Here's the new diagnostic

Reply:
I think the diagnostic should also include a mention of the consequence of the "too large" condition so the user knows what action to take—they just have to provide a default, right?

Reply:
This sounds good, along with what Joe said. We want the user to know immediately what to do next, even if there's some grumbling involved.

Reply:
If it doesn't require too much SourceLoc gymnastics, can we have a fixit insert `default: <#fatalError()#>` or something like that?